kolibri/plugins/learn/assets/src/learn.js

```diff
@@ -1,51 +1,19 @@
-'use strict';
-
-var _ = require('lodash');
-var logging = require('loglevel');
-
-// This is aliased for your convenience!
-var KolibriModule = require('kolibri_module');
-
-var LearnModule = KolibriModule.extend({
-
-    start: function(plugin) {
```

Comment:
We should probably have a method definable for all KolibriModules call `onRegister' or somesuch, that automatically fires when that plugin has finished registering. That way we can define a default point where plugins know that they are ready in the page to do stuff.